### ISSUE TITLE
shrink source data to help with memory issue on Ubuntu CI runners?

### ISSUE THREAD

**Issue by NickleDave** [NickleDave/vak#349]
see #341 

one possible cause is the source data alone could be taking up a lot of space

basic idea, use smaller training / test sets

we really do not care about accuracy or performance on usual CI runs, just need to test we didn't break anything.
Later can add "benchmark" functionality if needed.

**Comment by NickleDave** [comment]
Looking at this now.

Starting with `./tests/data_for_tests/source`
```
$ du -h --max-depth=1 .
88M	./spect_mat_annot_yarden
476M	./audio_cbin_annot_notmat
4.2G	./audio_wav_annot_textgrid
87M	./audio_wav_annot_koumura
```

- 4.2 GB (!) in the source data comes from the Cassin's Vireo dataset. 
  - and I'm not even sure how much we use this data?
  - yes, after investigating: we use the (gigantic!) `.WAV` files only to test that one function is case-insensitive
  - and we don't even use the textgrids
- next biggest is from audio cbin annot notmat, at ~500 M
  - should determine how many of these we really need just to run tests

**Comment by NickleDave** [comment]
fixed by #360
